### PR TITLE
On the tenant admin page, display organization name if there is no logoUrl

### PR DESCRIPTION
**Pull request by ksk100**
Small check added to organization admin page to print the organization's name if the logoUrl is blank.

### CODE REVIEW

File: AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Tenant/Index.cshtml [hunk 1]
```diff
@@ -24,7 +24,15 @@
             @foreach (var item in Model) {
                 <tr>
                     <td>
+                        @if (String.IsNullOrEmpty(item.LogoUrl))
```
Comment (mheggeseth) [inline AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Tenant/Index.cshtml]: You wouldn't have to repeat the a tag if you put the @if inside it.

Comment (ksk100) [inline AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Tenant/Index.cshtml]: Indeed.  I find it's sometimes clearer to write it out as I did it there, though, and it avoids potential problems with spurious spaces being appended to the start/end of the text inside the A element (after, say, the editor does some automatic indenting).  (Razor's shorthand `@( test ? "opt 1" : "opt 2")` isn't usable here because one of the options is an HTML tag).

More than happy to change it though; I'd also be interested to learn if there are any established best practices on this topic.  I've never come across any.

Comment (dpaquette) [inline AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Tenant/Index.cshtml]: I think the if statement should be inside the a tag.

Also don't forget a the alt attribute on the logo `alt="@item.Name"`

Comment (ksk100) [inline AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Tenant/Index.cshtml]: Okay.  I'll close this PR.

Comment (dpaquette) [inline AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Tenant/Index.cshtml]: You don't need to close the PR. Just make the change and commit to your branch.

Comment (mheggeseth) [inline AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Tenant/Index.cshtml]: Couldn't you avoid the errant spaces by wrapping `@item.Name` in a `<span>`?

Comment (ksk100) [inline AllReadyApp/Web-App/AllReady/Areas/Admin/Views/Tenant/Index.cshtml]: Ah ok; my mistake, I thought I needed to do a different PR.

**Comment by dpaquette** [comment]
:shipit:

**Comment by ksk100** [comment]
Closes #120 and closes #122

**Comment by dpaquette** [comment]
Looks good.
